q2_feature_table/tests/test_summarize.py

```diff
@@ -516,5 +518,33 @@ def test_vega_spec_nandling(self):
         self.assertEqual(spec['data'][0]['values'], exp)
 
 
+class TabulateSampleFrequencyTests(TestCase):
+
+    def test_basic_case(self):
+        table = biom.Table(np.array([[25, 25, 25], [25, 25, 25]]),
+                           ['O1', 'O2'],
+                           ['S1', 'S2', 'S3'])
+        metadata = tabulate_sample_frequencies(table).to_dataframe()
+
+        self.assertTrue('Frequency' in metadata.columns)
+        self.assertTrue('S1' in metadata.index)
+        self.assertTrue('S2' in metadata.index)
+        self.assertTrue('S3' in metadata.index)
+        self.assertTrue('50.0' in str(metadata))
+
+
+class TabulateFeatureFrequencyTests(TestCase):
+
+    def test_basic_case(self):
+        table = biom.Table(np.array([[25, 25, 25], [25, 25, 25]]),
+                           ['O1', 'O2'],
+                           ['S1', 'S2', 'S3'])
+        metadata = tabulate_feature_frequencies(table).to_dataframe()
+        self.assertTrue('Frequency' in metadata.columns)
+        self.assertTrue('O1' in metadata.index)
+        self.assertTrue('O2' in metadata.index)
+        self.assertTrue('75.0' in str(metadata))
```

Comment:
Here same as above from @gregcaporaso. Create the dataframe you expect to see. Something like

```
exp = pd.Dataframe("whatever data")
obs = # The dataframe you actually received as a result
pd.teseting.assert_frames_equal(exp, obs)
```

Documentation on assert_frames_equal [here](https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_frame_equal.html)

Reply:
Sounds good, I'll create this right now